core/src/Kokkos_ScratchSpace.hpp

```diff
@@ -55,7 +54,7 @@ class ScratchMemorySpace {
   mutable int m_offset        = 0;
   mutable int m_default_level = 0;
 
-  enum { MASK = ALIGN - 1 };  // Alignment used by View::shmem_size
+  constexpr static int DEFAULT_ALIGNMENT_MASK = ALIGN - 1;
```

Comment:
This one is also only used in `align()` and can thus be deprecated as well.
```suggestion
#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
  KOKKOS_DEPRECATED constexpr static int DEFAULT_ALIGNMENT_MASK = ALIGN - 1;
#endif
```

Reply:
its a private variable so I don't think we need to add deprecation?

Reply:
Sure, then just guard it so we don't forget to remove it later.